egs/wsj/asr1/conf/train_multispkr_transformer.yaml

```diff
@@ -0,0 +1,45 @@
+# network architecture
+# encoder related
+elayers-sd: 4
+elayers: 8
+eunits: 2048
+# decoder related
+dlayers: 6
+dunits: 2048
+# attention related
+adim: 256
+aheads: 4
+
+# hybrid CTC/attention
+mtlalpha: 0.2
+
+# label smoothing
+lsm-weight: 0.1
+
+# minibatch related
+batch-size: 16
+maxlen-in: 512  # if input length  > maxlen-in, batchsize is automatically reduced
+maxlen-out: 150 # if output length > maxlen-out, batchsize is automatically reduced
+
+# optimization related
+sortagrad: 0 # Feed samples from shortest to longest ; -1: enabled for all epochs, 0: disabled, other: enabled for 'other' epochs
+opt: noam
+accum-grad: 2
```

Comment:
I feel that `accum-grad` is a bit too small.
Maybe it is OK, but just a note.

Reply:
Yes, maybe it is too small. Later, I'll try to set `accum-grad` larger. Anyway, transformer with `accum-grad=2` shows improvement against RNN models.